helm-charts/values.yaml

```diff
@@ -77,3 +109,66 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+redis:
+  # Please see all available overrides at https://github.com/helm/charts/tree/master/stable/redis
+  # redis.enabled -- Flag to control whether to deploy Redis
+  enabled: true
+  # redis.fullnameOverride -- Name override for the redis deployment
+  fullnameOverride: "redis"
```

Comment:
Shouldn't this be `redis-headless` as defined above in `ordino.redis.host`?

Reply:
that is just a name override for the deployment which ensures that the headless service will be called redis-headless so that we can easily connect to it and have it preconfigured.